apps/web/pages/[user].tsx

```diff
@@ -77,7 +78,7 @@ export default function User(props: inferSSRProps<typeof getServerSideProps> & E
                 size="sm"
                 items={props.users.map((user) => ({
                   alt: user.name || "",
-                  image: user.avatar || "",
+                  image: `${WEBAPP_URL}/${user.username}/avatar.png`,
```

Comment:
I think user.avatar can be a base64 image, does the avatar endpoint takes care or that to be able to use it as the only way to access an avatar? @Udit-takkar

Reply:
@leog yeah this endpoint takes cares of  everything https://github.com/calcom/cal.com/blob/main/apps/web/pages/api/user/avatar.ts#L65

Reply:
Gotcha

Reply:
I have some observations here. Can we make it so `user.avatar` returns the full formed `${WEBAPP_URL}/${user.username}/avatar.png` instead? WE might benefit by not sending the whole base64 URI in these responses.

Reply:
@zomars what do you mean by returning the full formed ${WEBAPP_URL}/${user.username}/avatar.png ?

currently when we make request to the endpoint /api/user/avatar we fetch the user data from database and user.avatar could be base64 image which is then decoded and sent back.

Reply:
Let's just avoid sending the full base64 image in the responses